package.json

```diff
@@ -18,7 +18,7 @@
 		"@guardian/atom-renderer": "^2.0.0",
 		"@guardian/automat-modules": "^0.3.8",
 		"@guardian/braze-components": "^15.0.1",
-		"@guardian/commercial": "11.15.0",
+		"@guardian/commercial": "^11.16.0",
```

Comment:
Is the addition of the caret intentional? I'm not against it's inclusion, just wanted to check.